tracks/csharp/exercises/resistor-color/mentoring.md

```diff
@@ -0,0 +1,97 @@
+### Reasonable solutions
+#### Array Based
+``` 
+public static class ResistorColor
+{
+    private static readonly string[] resistorColors = { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
+
+    public static int ColorCode(string color) => System.Array.IndexOf(resistorColors, color);
+
+    public static string[] Colors() => resistorColors;
+}
+```
+#### Dictionary Based
+Based on Horatiu's solution
+```
+﻿using System.Collections.Generic;
+ using System.Collections.ObjectModel;
+ using System.Linq;
+  
+ public static class ResistorColor
+ {
+     private static ReadOnlyDictionary<string, int> colors = new ReadOnlyDictionary<string, int>(new Dictionary<string, int>
+     {
+         {"black", 0},
+         {"brown", 1},
+         {"red" , 2},
+         {"orange", 3},
+         {"yellow", 4},
+         {"green", 5},
+         {"blue", 6},
+         {"violet", 7},
+         {"grey", 8},
+         {"white", 9}
+     });
+ 
+     public static int ColorCode(string color)
+     {
+         return colors[color];
+     }
+ 
+     public static string[] Colors()
+     {
+         return colors.Keys.ToArray();
+     }
+ }
+```
+#### Enum Based
+```
+using System;
+
+public static class ResistorColor
+{
+    enum Color
+    {
+        black,
+        brown,
+        red,
+        orange,
+        yellow,
+        green,
+        blue,
+        violet,
+        grey,
+        white
+        
+    }
+    public static int ColorCode(string color)
+    {
+        return (int)Enum.Parse(typeof(Color), color);
+    }
+
+    public static string[] Colors()
+    {
+        return Enum.GetNames(typeof(Color));        
+    }
+}
+```
+### Common suggestions
+* Enum version: If users will be inputting or reading-out number values for the
+resistors than it might be more expressive to assign the numbers to
```

Comment:
```suggestion
resistors then it's more expressive to assign the numbers to
```
I suggest concise wording with less conditionals. Fixed `than` -> `then`. I suggest that limit the amount of "doubt" (might be, could be) in mentor notes.